server_spidercast/src/MCC_RUM/src/rmmCutils.c

```diff
@@ -990,9 +990,9 @@ int rmm_get_thread_stacksize(pthread_t thread_id, int *size)
 char *upper(char *str)
 {
   char *ip ;
-  for ( ip=str ; !isEOL(*ip) ; ip++ )
+  for ( ip=str ; !isEOL(*ip) ; ip++ ){
     if ( *ip >= 'a' && *ip <= 'z' ) *ip = 'A' + (*ip-'a') ;
-
+  }
     return str ;
```

Comment:
can you unindent the return line so it's in line with the for

Reply:
Okay will do that change.